README.md

```diff
@@ -1,23 +1,35 @@
 <p align="center">
   <img src="https://i.imgur.com/22sf4x7.png" />
 </p>
-<p align="center"><strong>A lightweight cloud orchestrator in Rust</strong></p>
-<p align="center">❗ This project is experimental and should NOT be used in production. ❗</p>
+<img src="https://img.shields.io/github/actions/workflow/status/rik-org/rik/.github/workflows/rust.yml?branch=main&style=for-the-badge" />
 
-<div>
-<img src="https://img.shields.io/badge/Rik-rust-orange?style=for-the-badge&logo=appveyor" />
-<img src="https://img.shields.io/github/workflow/status/thomasgouveia/rik/RIK%20CI?style=for-the-badge" />
-<img alt="Discord" src="https://img.shields.io/discord/863020591984148502?style=for-the-badge">
-</div>
+## What is RIK ?
 
-### Contributing
+RIK (pronounced /rɪk/) is an experimental workload orchestrator written in Rust that aims to be able to natively schedule **containers** and **virtual machines** workloads. RIK stands for **R**ust **i**n **K**ubernetes.
 
-**RIK** is open-source and contributions are welcome. Please read the [CONTRIBUTING.md](CONTRIBUTING.md) for more information on how to contribute to this project.
+### Motivation
 
-## Troubleshooting
+Cloud adoption is growing every year. A lot of technologies were created to solve issues it brings on. One of the most popular technology when dealing with cloud is **containers**. As stated by the [CNCF annual survey of 2022](https://www.cncf.io/reports/cncf-annual-survey-2022/), containers are the new normal. A lot of companies are using containers in production to run their applications because of the portability, scalability and other benefits they offer. But in a world where elasticity is the key for building cost-efficient resilient applications, some tools must be used to automate management of the containers. This is where orchestrators like [Kubernetes](https://github.com/kubernetes/kubernetes), [HashiCorp Nomad](https://github.com/hashicorp/nomad) or others comes into play. 
 
-**`cargo build` fails because cannot build `openssl-sys`**
+The goal of RIK is not to replace Kubernetes or similar. It is a project mainly for educational purposes, and we want to provide a lightweight and simple code base to **help people understand underlying concepts of orchestrators**, or simply **to learn Rust** on a project related to a day-to-day issue: **orchestrate cloud applications**. 
```

Comment:
To replace ? :eyes:  I don't think that rik purpose is to replace K8s.
More mimic k8s i think

Reply:
What do you mean? 

I want to say that we don't want to replace it, but create something similar that is easier to understand for people beginning in the domain.

Reply:
ah my bad i didn't read the "not"

Reply:
LGTM